docs/current/labs/project-zenith.md

```diff
@@ -175,7 +71,7 @@ mkdir potato/
 cd potato/
 
 # initialize Dagger module
-# NOTE: currently Go is the only supported SDK.
+# NOTE: currently only the Go SDK implements `dagger mod init`, but Python modules also work and are gaining features rapidly. Node.js modules are not yet available, but under development.
```

Comment:
`dagger mod init --sdk=python` does work, you just have to create a `pyproject.toml` and `src/main.py` first (it doesn't make those for you yet).

Just saw this was merged in the middle of writing my comment 😄 I think this is fine as is; we're meeting w/ Vikram tomorrow to talk more about docs, can easily follow-up more after that

Reply:
I actually implemented this yesterday, just need to push! 🙂

Reply:
- https://github.com/dagger/dagger/pull/5959